storage/scylla_table.c

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include "scylla_table.h"
+#include "common/model/transaction.h"
+#include "utils/logger.h"
+
+static logger_id_t logger_id;
+
+void scylla_table_logger_init() { logger_id = logger_helper_enable("scylla_table", LOGGER_DEBUG, true); }
+
+int scylla_table_logger_release() {
+  logger_helper_release(logger_id);
+  if (logger_helper_destroy() != RC_OK) {
+    log_critical(logger_id, "%s.\n", "scylla_table");
+    return EXIT_FAILURE;
+  }
+  return 0;
+}
+
+struct db_identity_s {
+  CassUuid uuid;
+  cass_int8_t status;
+  cass_byte_t hash[NUM_FLEX_TRITS_BUNDLE];
+};
+
+status_t db_new_identity(db_identity_t** obj) {
+  *obj = (db_identity_t*)malloc(sizeof(struct db_identity_s));
+  if (NULL == *obj) {
+    ta_log_error("SC_STORAGE_OOM\n");
+    return SC_STORAGE_OOM;
+  }
+
+  return SC_OK;
+}
+
+void db_free_identity(db_identity_t** obj) {
+  free(*obj);
+  *obj = NULL;
+}
+
+status_t db_set_identity_uuid(db_identity_t* obj, CassUuid uuid) {
```

Comment:
I didn't have idea about the necessity of UUID? Can you explain and add comments inside source code?

Reply:
After reattaching to the Tangle, the Transaction hash will change. Users from Alfred can not find the transaction by the original hash. UUID is the substitution for Users to query.
I am considering using UUID or auto-increment keys to manage it.
The flow is that TAs will call an API provided in ta-scylla-backend which will insert the transaction hash, a new genarated UUID and status(PENDING) into `db_identity_table` after successfully send a new transaction to IRI.

Reply:
The auto-increment keys might help. We don't really require specific storage for UUID.

Reply:
If using auto-increment keys, I think it might be better that Alfred provides the keys. Since there are multiple TAs are parallelly running, it is hard to synchronize the number of keys.

Reply:
It is not "parallelly". It is "concurrently".

Reply:
Got it!
So I will remove UUID related code and relay on integer keys given by Alfred.